package/src/types/types.ts

```diff
@@ -3,7 +3,7 @@ import type { ExtendableGenerics, LiteralStringForUnion } from 'stream-chat';
 export type Asset = {
   duration: number | null;
   filename: string;
-  fileSize: string;
+  fileSize: number | undefined;
```

Comment:
`fileSize` is undefined in expo 
and it is always a number

Reply:
```suggestion
  fileSize?: number;
```